packages/common/src/groups/_internal/computeProps.ts

```diff
@@ -62,6 +65,8 @@ export function computeProps(
     group.userMembership?.memberType === "admin";
   hubGroup.canDelete = hubGroup.canEdit;
 
+  hubGroup.links = computeLinks(group, requestOptions);
```

Comment:
Do we want to expose links to the entity? If so, we might want to do the same for other item-based entities as well.

Reply:
I do believe this is the intention - Caleb merged [this](https://github.com/Esri/hub.js/pull/1199) PR a couple weeks ago that adds an optional `links` hash to the `IHubEntityBase` - so all hub entities should inherit this property. In his PR, Caleb hooked up these links for Hub Sites, Pages, and Content. This PR is to similarly hook it up for _our_ entities - Projects, Initiatives, and Groups

Reply:
Ah I see, this makes sense